internal/pkg/agent/application/monitoring/v1_monitor.go

```diff
@@ -424,6 +424,13 @@ func (b *BeatsMonitor) injectLogsInput(cfg map[string]interface{}, components []
 								"to":   "data_stream.dataset_original",
 							},
 						},
+						"when": map[string]any{
```

Comment:
do we anticipate something to have written in the target `data_stream.dataset_original` field? As far as I can this condition missing isn't the reason of the reported issue, right?

Reply:
> As far as I can tell this condition missing isn't the reason of the reported issue, right?

You're right, it's not on the reported issue.

Honestly, I don't know by heart all the possible ways `data_stream.dataset_original` could be set. So out of extra caution I added the `when` to all `copy_fields` processors to avoid any of them flooding the debug logs.

Reply:
Done on 7f3b86f3db